File: src/QuizAppCSharp/QuizApp/Model/Topic.cs
using System;
using System.Collections.Generic;
using System.Collections.Specialized;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Web;

namespace QuizApp
{
    public class Topic
    {

        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public HashSet<Question> Questions { get; set; }

        public Topic(int id, string name, string description)
        {
            Id = id;
            Name = name;
            Description = description;
        }

    }
}

File: src/main/java/com/gotik/quizgeneration/Model/Topic.java
package com.gotik.quizgeneration.Model;

import java.util.HashSet;
import java.util.Set;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;
@Builder
@Getter
@Setter
public class Topic {
    private int id;
    private String name;
    private String description;
    @Builder.Default
    private Set<Question> questions = new HashSet<>();

}
